src/tables.jl

```diff
@@ -1,20 +1,20 @@
 Tables.istable(::Type{<:Result}) = true
 Tables.rowaccess(::Type{<:Result}) = true
-Tables.rows(r::Result) = r
+Tables.rows(jl_result::Result) = jl_result
 
-Base.eltype(r::Result) = Row
-Base.length(r::Result) = num_rows(r)
+Base.eltype(jl_result::Result) = Row
+Base.length(jl_result::Result) = num_rows(jl_result)
 
-function Tables.schema(r::Result)
-    types = map(r.not_null, column_types(r)) do not_null, col_type
+function Tables.schema(jl_result::Result)
+    types = map(jl_result.not_null, column_types(jl_result)) do not_null, col_type
         not_null ? col_type : Union{col_type, Missing}
     end
-    return Tables.Schema(types, map(Symbol, column_names(r)))
+    return Tables.Schema(map(Symbol, column_names(jl_result)), types)
```

Comment:
Was this the only thing that was broken?

Reply:
Line 17 was also broken as it referenced `result` instead of `r`